README.md

```diff
@@ -0,0 +1,78 @@
+[![License badge](https://img.shields.io/github/license/FIWARE/context.Orion-LD.svg)](https://opensource.org/licenses/AGPL-3.0)
+[![Container Repository on Quay](https://img.shields.io/badge/quay.io-eas-green "Container Repository on Quay")](https://quay.io/repository/wi_stefan/endpoint-configuration-service?tab=tags)
```

Comment:
Will these links need amending after moving to the FIWARE org?

Reply:
Yes